library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/CirclePromptBackground.java

```diff
@@ -174,12 +174,11 @@ else if (x3 > focalLeft && x3 < focalRight)
         else
         {
             mBasePosition.set(focalCentreX, focalCentreY);
-            // Calculate the furthest distance from the a focal side to a text side.
-            final float length = Math.abs(
-                    (textBounds.left < focalBounds.left ?
-                            textBounds.left - textPadding
-                            : textBounds.right + textPadding)
-                    - focalCentreX);
+           // Calculate the furthest distance from the center based on the text size.
+            final float length = Math.max(
+                    Math.abs(textBounds.right - focalBounds.centerX()),
+                    Math.abs(textBounds.left - focalBounds.centerX())
+            ) + textPadding + focalBounds.width() / 2f
```

Comment:
The `focalBounds.width() / 2f` doesn't make mathematical sense here unless I am mistaken. The `textBounds.right - focalBounds.centerX()` gets the distance from the centre of the view that is targeted (focal) to the text edge. Adding on half the focal width is just adding an arbitrary number to increase the background size